example/ionic-angular/README.md

```diff
@@ -0,0 +1,69 @@
+# Instructions for using this test app
+
+## Install dependencies
+`npm install`
+
+## Build for native
+`ionic build`
+
+## Sync resources
+`npx cap sync`
+
+## Open emulator
+`npx cap open android`
+
+
+# General Capacitor/Ionic Instructions
+
+## Add Capacitor to an existing Ionic project
+Full details can be found at capacitorjs.com/docs/getting-started/with-ionic
+1. `ionic integrations enable capacitor`
+2. `npx cap init [appName] [appId]`
+3. `ionic build`
+4. `npx cap add ios`
+5. `npx cap add android`
+6. `npx cap sync`
+
+## Add Sentry/Capacitor to your project
```

Comment:
Is this relevant to the sample? Or should this just go under the docs for general instructions on how to add it to an app?